_rest_config.php

```diff
@@ -77,7 +77,7 @@ static function Init()
     {
         if (!self::$IS_INITIALIZED) {
             self::setPaths();
-            self::$REST_FULL_URL = $_SERVER['REQUEST_SCHEME'] . "//" . $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI']; // @todo unsure here!
+            self::$REST_FULL_URL = $_SERVER['REQUEST_SCHEME'] . "//" . $_SERVER['SERVER_NAME'] . $_SERVER['REDIRECT_URL']; // @todo unsure here!
```

Comment:
Need to change this because the  `$_SERVER['REQUEST_URI']` comes with query string which can't be used in "fullUrl"
Example : ` "fullUrl": "http//localhost/openemr/apis/fhir/Encounter?_id=90b834ff-b521-488d-aa3e-8f60193db3ec/90b834ff-b521-488d-aa3e-8f60193db3ec"`

Another possible solution could be
`parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)`